R/calcAirPollEmiRef.R

```diff
@@ -0,0 +1,94 @@
+#' Calculate air pollutant emissions for a reference year, for use
+#' in combination with GAINS data at different sectoral aggregations
+#'
+#'
+#' @param baseyear year to take as a reference from CEDS, ignored for the EDGAR2005 LUC CO2 emissions
+#' @return magclass object
+#' @author Gabriel Abrahao
+#' @importFrom magclass getNames<- getYears<-
+
+calcAirPollEmiRef <- function(
```

Comment:
Please add documentation for parameters `subtype`, `outunits` and `namesformat`

Reply:
Consider dropping `outunits`, as it  always is "Mt/yr"

Reply:
Documented. `kt/yr` is not used now but I plan to, for harmonizing some emissions to CEDS later